sdk/formrecognizer/ai-form-recognizer/package.json

```diff
@@ -3,7 +3,7 @@
   "sdk-type": "client",
   "author": "Microsoft Corporation",
   "description": "An isomorphic client library for the Azure Form Recognizer service.",
-  "version": "3.0.1",
+  "version": "3.1.0-preview.1",
```

Comment:
do we use `-beta.1`, or still keep using `-preview.1` for libraries that already ship with `preview.*` versions?

Reply:
Oh good point, I haven't shipped a beta yet and forgot.